doc/user_guide/configuration/all-options.rst

```diff
@@ -233,7 +233,7 @@ Standard Checkers
 
    confidence = ["HIGH", "CONTROL_FLOW", "INFERENCE", "INFERENCE_FAILURE", "UNDEFINED"]
 
-   disable = ["raw-checker-failed", "bad-inline-option", "locally-disabled", "file-ignored", "suppressed-message", "useless-suppression", "deprecated-pragma", "use-symbolic-message-instead", "consider-using-augmented-assign"]
+   disable = ["consider-using-augmented-assign"]
```

Comment:
Quick question: from where did this change arise?

When I `--generate-toml-config` on 2.17.5 I get:

```
disable = ["raw-checker-failed", "bad-inline-option", "locally-disabled", "file-ignored", "suppressed-message", "deprecated-pragma", "consider-using-augmented-assign", "attribute-defined-outside-init", "invalid-name", "missing-module-docstring", "missing-class-docstring", "missing-function-docstring", "protected-access", "too-few-public-methods", "line-too-long", "too-many-lines", "trailing-whitespace", "missing-final-newline", "trailing-newlines", "bad-indentation", "unnecessary-semicolon", "multiple-statements", "superfluous-parens", "mixed-line-endings", "unexpected-line-ending-format", "fixme"]
```

Reply:
Applied ``tox -e docs`` (probably from tbump)

Reply:
Interesting. So there's a bug somewhere here, then?

Reply:
Maybe it's based on the local pylintrc ?